charts/urbanstack-shared-app/templates/deployment.yaml

```
@@ -0,0 +1,99 @@
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "urbanstack-shared-app.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels: {{- include "urbanstack-shared-app.labels" . | nindent 8 }}
+    spec:
+      {{- if and .Values.registryAuth.username .Values.registryAuth.password }}
+      imagePullSecrets:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)